redash/query_runner/__init__.py

```diff
@@ -73,6 +73,23 @@ def annotate_query(cls):
     def configuration_schema(cls):
         return {}
 
+    def get_data_source_version(self):
+        if self.data_source_version_query is None:
+            raise NotImplementedError
+        data, error = self.run_query(self.data_source_version_query, None)
+        version = json.loads(data)['rows'][0]['version']
```

Comment:
This line seems like a place where `KeyError`s could happen easily. Should there be some try/except clause?